src/lib/RestAPI/arrivalsAndDeparturesForStop.js

```diff
@@ -1,17 +1,20 @@
 import { error, json } from '@sveltejs/kit';
-
 import { PUBLIC_OBA_SERVER_URL as baseURL } from '$env/static/public';
 import { PRIVATE_OBA_API_KEY as apiKey } from '$env/static/private';
 
+import onebusaway from 'onebusaway-sdk';
+
+const oba = new onebusaway({
+	baseURL,
+	apiKey
+});
+
 export default async function (stopID) {
-	const apiURL = `${baseURL}/api/where/arrivals-and-departures-for-stop/${stopID}.json?key=${apiKey}`;
-	const response = await fetch(apiURL);
+	const response = await oba.arrivalAndDeparture.list(stopID);
 
-	if (!response.ok) {
+	if (response.code !== 200) {
 		error(500, 'Unable to fetch arrivals-and-departures-for-stop.');
```

Comment:
does the `error()` method here function function like an exception? i.e. does it interrupt control flow?

Reply:
![image](https://github.com/user-attachments/assets/809f6d3f-3c81-492e-82aa-6f5ff1d246b0)

I don't know, I try it now, it seems not interrupt the flow, but when I enter stops route endpoint the error message appears in the top

![image](https://github.com/user-attachments/assets/b2e1d820-8f55-4ab8-af3c-d50db441fd3d)

https://learn.svelte.dev/tutorial/error-basics

but according to svelte docs the `expected error message is shown to the user`.